rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -1666,43 +1712,58 @@ rocalColorTwistFixed(
     return output;
 }
 
-RocalImage
-ROCAL_API_CALL rocalCropMirrorNormalize(RocalContext p_context, RocalImage p_input, unsigned crop_depth, unsigned crop_height,
-                                    unsigned crop_width, float start_x, float start_y, float start_z, std::vector<float> &mean,
-                                    std::vector<float> &std_dev, bool is_output, RocalIntParam p_mirror)
+RocalTensor
+ROCAL_API_CALL rocalCropMirrorNormalize(RocalContext p_context, RocalTensor p_input, RocalTensorLayout rocal_tensor_layout,
```

Comment:
can we please move rocalCropMirrorNormalize(  to new line for maintaining same style as with other functions?